packages/common/src/store/notifications/types.ts

```diff
@@ -260,6 +264,18 @@ export type DiscoveryUSDCPurchaseNotificationAction = {
   extra_amount: StringUSDC
 }
 
+export type DiscoveryRequestManagerNotificationAction = {
+  user_id: string
```

Comment:
Just one thing I was a little confused about with these notifications.
How does `user_id` relate to `grantee_user_id` and `grantee_address` in each case?
Is `user_id` going to be the user who requested the manager for this notification and the user who approve the request in `ApproveManagerRequestNotification` case?

Reply:
good q - in both notifs, user_id is the person who requested the manager, i.e. `user_id` is always the person who is being managed (this matches the meaning of `user_id` in the Grants table).